docs/claimFaucet.md

```diff
@@ -0,0 +1,163 @@
+
+If you’re new to Godwoken, it’s normal that your `L1 Balance` and `L1 Balance` are both `0`. You can follow the tutorials below to claim free assets to your layer1/layer2 accounts. 
+
+:::tip
+For more details on Godwoken Bridge, please visit: [Deposit & Fast Withdrawal to Godwoken v1](fastWithdrawal.md).
+:::
+
+## Claim Layer1 assets
+
+### Claim CKB
+
+On the CKB layer1 testnet, you can claim CKB using the Nervos Aggron Faucet, a official faucet site provided by Nervos. On Nervos Aggron Faucet, each layer1 address can claim 10,000 CKB once every 24 hours.
+
+Follow these steps to claim CKB to your layer1 account:
+
+1. Copy your `L1 Wallet Address` on [Godwoken Bridge](https://testnet.bridge.godwoken.io/)
+2. Open [Nervos Aggron Faucet](https://faucet.nervos.org/)
+3. Paste `L1 Wallet Address` in the page input
+4. Click “Claim” button next to the page input
+
+Usually you should wait for a minute for the process to complete.
+
+If you like to check the status of your faucet records, there’s a “Search Address” filter input on the page, you can paste your `L1 Wallet Address` in the input, and then click “Enter” on your keyboard to search for claim records of your layer1 address.
+
+:::tip
+For more info, Nervos Aggron Faucet accepts all kinds of valid CKB address, so you can actually claim pCKB for your layer2 account using Nervos Aggron Faucet. For this part of tutorial, please visit this part of content: [Claim pCKB](#claim-pckb).
+:::
+
+### Claim layer1 tokens
+
+TTKN is a sUDT (Simple User Defined Tokens) on Testnet of CKB layer1, if you want to test the functionality of depositing sUDT from layer1 to layer2, you can claim TTKN on Godwoken Bridge for free.
+
+You can claim TTKN by following these steps:
+1. Open and connect to [Godwoken Bridge](https://testnet.bridge.godwoken.io/)
+2. Click the menu button on the header of page
+3. Click “Get 1,000 Test Token(TTKN) on L1” button
+
+When it’s done, you should be able to see your TTKN layer1 balance on the deposit page of Godwoken Bridge.
+
+## Claim Layer2 assets
+
+### Claim pCKB
+
+If you want to claim pCKB to your layer2 account, there are two recommended methods.
+
+The first method is if you only need a small amount of pCKB, you can visit [Test pCKB Faucet](https://faucet.godwoken.com/), and then click the “request 10 pCKB from faucet” button to claim 10 pCKB to your layer2 account, on the way it will ask you to connect to your wallet, you should just follow the instructions and the whole process will complete soon.
+
+However, if you want to claim a large amount of pCKB, the second method is more recommended: [Nervos Aggron Faucet](https://faucet.nervos.org/). 
+
+Nervos Aggron Faucet is an official faucet site provided by Nervos. On Nervos Aggron Faucet, each layer1 address can claim 10,000 CKB once every 24 hours. The site only provides CKB, but you can claim funds using your `L1 Deposit Address`, so it's like Nervos Aggron Faucet depositing capacity to your layer2 account. 
```

Comment:
```suggestion
```

Reply:
I think we should keep this part of content, because readers might skip the #claim-ckb part.

Reply:
I think we should keep this section as it explains a bit about Nervos faucets. I've edited this part and tried to make it more reasonable, please take a look.